build/sources.props

```diff
@@ -27,6 +27,11 @@
       $(RestoreSources);
       https://dotnet.myget.org/F/aspnetcore-dev/api/v3/index.json;
     </RestoreSources>
+    <!-- TODO remove this once we move Microsoft.AspNetCore.Blazor.Mono to a non-myget feed -->
+    <RestoreSources>
+      $(RestoreSources);
+      https://dotnet.myget.org/F/blazor-dev/api/v3/index.json;
+    </RestoreSources>
```

Comment:
Flagging this to @natemcmaster @dougbu 

Looks like Microsoft.AspNetCore.Blazor.Mono doesn't have a new home on dotnetfeed.blob.core.windows.net yet.

Since this follows the same pattern as for other un-rehomed packages above, it looks reasonable to unblock ourselves by adding this `blazor-dev` source here. Not sure which feed it should ultimately be moved to, or whether it needs a new feed of its own.

Reply:
I will address this as a part of aspnet/AspNetCore-Internal
#2360. We've been moving away from myget feeds due to perf and reliability reasons, so this will get updated with a static feed instead.